src/main/java/uk/gov/justice/hmpps/prison/api/resource/BookingResource.java

```diff
@@ -142,7 +142,6 @@ public class BookingResource {
         @ApiResponse(responseCode = "500", description = "Unrecoverable error occurred whilst processing request.", content = {@Content(mediaType = "application/json", schema = @Schema(implementation = ErrorResponse.class))})})
     @Operation(summary = "Prisoners Booking Summary ", description = "Returns data that is available to the users caseload privileges, at least one attribute of a prisonId, bookingId or offenderNo must be specified")
     @GetMapping("/v2")
-    @VerifyBookingAccess(overrideRoles = {"VIEW_PRISONER_DATA"})
```

Comment:
This wasn't doing anything - for verifyBookingAccess it needs to pass in  "bookingId".  
bookingIds are not mandatory and "bookingIds" is passed in - a list